osbs/build/build_requestv2.py

```diff
@@ -644,6 +644,10 @@ def __init__(self, build_json_store, outer_template=None):
         self.user_params = SourceContainerUserParams(self.build_json_store)
 
     def render(self, validate=True):
+        koji_task_id = self.user_params.koji_task_id.value
+        if koji_task_id is not None:
+            self.set_label('koji-task-id', str(koji_task_id))
```

Comment:
Should we add a test for this?

Reply:
moved to common, is covered by test for regular builds

Reply:
win-win situation :D